eslint.config.js

```diff
@@ -1,5 +1,167 @@
+                (arg, i, arr) => {
+                  let prev = i === 0 ? startLine : arr[i - 1].loc.end.line
+                  if (arg.loc.start.line === prev) {
+                    actions.push(fixer.insertTextBefore(arg, '\n'))
+                  }
+                }
+              )
+              let lastArg = node.arguments[node.arguments.length - 1]
+              if (lastArg.loc.start.line === endLine) {
+                actions.push(fixer.insertTextAfter(lastArg, '\n'))
+              }
+              return actions
+            }
+          })
+        }
+      })
+    }
+  }
+}
 
 export default [
-    ...blumilkDefault,
+  ...blumilkDefault,
+  {
+    plugins: {
+      '@stylistic': stylisticPlugin,
+      'import': importPlugin,
+      'custom': custom,
+    },
+    rules: {
+      'quotes': 'off',
+      'indent': 'off',
+      'semi': 'off',
+      'comma-dangle': 'off',
+      'eol-last': 'off',
+      '@typescript-eslint/no-var-requires': 'off',
+      '@typescript-eslint/prefer-ts-expect-error': 'off',
+      '@typescript-eslint/member-delimiter-style': 'off',
+
+      'custom/function-multiline-arguments': 'warn',
+
+      'no-console': 'warn',
+      'no-empty': 'warn',
+      'arrow-body-style': ['warn', 'as-needed'],
+
+      '@stylistic/quotes': ['warn', 'single'],
+      '@stylistic/indent': ['warn', 2],
+      '@stylistic/semi': ['warn', 'never'],
+      '@stylistic/comma-dangle': ['warn', 'always-multiline'],
+      '@stylistic/eol-last': ['warn', 'always'],
+      '@stylistic/no-multiple-empty-lines': ['warn', { max: 2, maxBOF: 0, maxEOF: 0 }],
+      '@stylistic/member-delimiter-style': ['warn', {
+        multiline: { delimiter: 'none' },
+        singleline: { delimiter: 'comma', requireLast: false },
+      }],
+      '@stylistic/comma-spacing': ['warn', { 'after': true }],
+      '@stylistic/quote-props': ['warn', 'as-needed'],
+      '@stylistic/rest-spread-spacing': ['warn', 'never'],
+      '@stylistic/array-bracket-spacing': ['warn', 'never'],
+      '@stylistic/array-bracket-newline': ['warn', { multiline: true }],
+      '@stylistic/array-element-newline': ['warn', 'consistent'],
+      '@stylistic/object-curly-spacing': ['warn', 'always'],
+      '@stylistic/object-curly-newline': ['warn', { multiline: true }],
+      '@stylistic/object-property-newline': ['warn', { allowMultiplePropertiesPerLine: true }],
+      '@stylistic/key-spacing': 'warn',
+      '@stylistic/type-annotation-spacing': 'warn',
+      '@stylistic/switch-colon-spacing': 'warn',
+      '@stylistic/implicit-arrow-linebreak': ['warn', 'beside'],
+      '@stylistic/arrow-spacing': 'warn',
+      '@stylistic/brace-style': ['warn', 'stroustrup'],
+      '@stylistic/function-paren-newline': ['warn', 'multiline-arguments'],
+      '@stylistic/space-in-parens': ['warn', 'never'],
+      '@stylistic/function-call-spacing': ['warn', 'never'],
+      '@stylistic/space-unary-ops': 'warn',
+
+      'import/first': 'error',
```

Comment:
You set so many rules here. Did you check this is unnecessary to set ? Maybe some rules are in imported config. Or when You override so many rules from basic imported config, maybe you don't want to use imported config.